ambari-project/pom.xml

```diff
@@ -32,7 +32,7 @@
     <jetty.version>8.1.19.v20160209</jetty.version>
     <checkstyle.version>6.19</checkstyle.version> <!-- last version that does not require Java 8 -->
     <checkstyle.skip>false</checkstyle.skip>
-    <fasterxml.jackson.version>2.9.4</fasterxml.jackson.version>
+    <fasterxml.jackson.version>2.9.8</fasterxml.jackson.version>
```

Comment:
Since we use the dependencyManagement section at ambari-project/pom.xml , how about moving the dependency definition here, and inheriting dependency version at all child projects
https://stackoverflow.com/a/38882340